<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录/注册</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_login">
      <attribute name="title">
       <string>登录</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_login">
       <item>
        <layout class="QFormLayout" name="formLayout_login">
         <item row="0" column="0">
          <widget class="QLabel" name="label_username_login">
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_username_login">
           <property name="maxLength">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_password_login">
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_password_login">
           <property name="maxLength">
            <number>40</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_login">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_login">
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_register">
      <attribute name="title">
       <string>注册</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_register">
       <item>
        <layout class="QFormLayout" name="formLayout_register">
         <item row="0" column="0">
          <widget class="QLabel" name="label_username_register">
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_username_register">
           <property name="maxLength">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_password_register">
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_password_register">
           <property name="maxLength">
            <number>40</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_confirm_password">
           <property name="text">
            <string>确认密码</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_confirm_password">
           <property name="maxLength">
            <number>40</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_hobbies">
         <property name="title">
          <string>喜好选择</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_hobbies">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_select">
            <item>
             <widget class="QCheckBox" name="cb_selectAll">
              <property name="text">
               <string>全选</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cb_deselectAll">
              <property name="text">
               <string>全不选</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_select">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_hobbies">
            <item row="0" column="0">
             <widget class="QCheckBox" name="cb_food">
              <property name="text">
               <string>美食</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="cb_funny">
              <property name="text">
               <string>搞笑</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="cb_ennegy">
              <property name="text">
               <string>正能量</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QCheckBox" name="cb_dance">
              <property name="text">
               <string>舞蹈</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="cb_music">
              <property name="text">
               <string>歌曲</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="cb_video">
              <property name="text">
               <string>影视</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QCheckBox" name="cb_outside">
              <property name="text">
               <string>户外</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QCheckBox" name="cb_edu">
              <property name="text">
               <string>教育</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_register">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_register_buttons">
         <item>
          <widget class="QPushButton" name="pb_register">
           <property name="text">
            <string>注册</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_clear">
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
